client/src/i18n.js

```diff
@@ -207,14 +214,13 @@ class i18n {
 
     // Get locale (e.g. 'en_US') from common name (e.g. 'en')
     // by looking at i18n.lang tables
-    let detectedLocale = null;
     if (rawLocale.length === 2) {
-      for (let compareLocale in i18n.lang) {
+      for (let compareLocale in this.lang) {
```

Comment:
The dictionaries are added to _this_ instance of `i18n`, not as a static property on the class.